src/components/Account.css

```diff
@@ -0,0 +1,28 @@
+body{
```

Comment:
Important to note that this is here for later. Our body isn't specified specifically for each page with an id so this will apply to all pages. Just to keep in mind in case any css changes on the other pages.